### PR TITLE
Create section in application connector for ServiceNow user guide + Add section for description off SN workflows

### PR DESCRIPTION
**Pull request by AlexFalessi**
The section "Communicate with ProActive" of the admin guide has been moved to the application connector section of the user guide and is now called "ServiceNow to ProActive". This has already been reviewed and validated.

Please review the new section "ProActive to ServiceNow"

### CODE REVIEW

File: src/docs/admin/references/ServiceNowReference.adoc [hunk 1]
```diff
@@ -19,8 +19,8 @@ It boils down to: +
 ==== Installation
 
 This section describes the available processes to set up the required files in ServiceNow to communicate
-with ProActive. The preferred installation process is dictated by the architecture of your application ecosystem
-in ServiceNow. Either ProActive's integration will have a dedicated ServiceNow application, or it will be combined with an existing one.
+with ProActive.The preferred installation process is dictated by the architecture of your application ecosystem
+in ServiceNow.Either ProActive's integration will have a dedicated ServiceNow application, or it will be combined with an existing one.
```
Comment (marounkoussaifi) [inline src/docs/admin/references/ServiceNowReference.adoc:23]: Why the space between the "." and the next character have to be removed?

File: src/docs/admin/references/ServiceNowReference.adoc [hunk 2]
```diff
@@ -101,461 +101,11 @@ image::servicenow-search-by-status.png[align="center"]
 Result should be empty.
 
 
-==== Communicate with ProActive
+==== Usage
 
-Once the application is installed, you are ready to integrate ProActive in your ServiceNow pipelines.
+Once the application imported in your ServiceNow instances, you are ready to communication both ways between ProActive and ServiceNow.
```
Comment (marounkoussaifi) [inline src/docs/admin/references/ServiceNowReference.adoc:106]: is imported

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 3]
```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
 This WorkFlow performs a GET request to an Informatica API associated to an application already created by the Informatica user.
 This workflow template supports the following authentication methods: a basic authentication using a username and a password, an advanced authentication using a JWT access token and finally an anonymous authentication where no authorization is required.
 The authentication method is determined based on the definition of the parameters when launching the WF.
+
+
+[#_servicenow]
+=== ServiceNow
+
+The ServiceNow connectors allows ProActive to communicate with ServiceNow instances to perform CRUD operations on their data as well as send emails.
```
Comment (marounkoussaifi) [inline src/docs/user/references/ApplicationConnectors.adoc:86]: allow instead of allows
as well as to send email

**Review by marounkoussaifi — APPROVED** [review]

**Comment by marounkoussaifi** [comment]
@AlexFalessi I left you some minor comments to see. Nothing is blocking. Nicely written documentation :)

**Comment by AlexFalessi** [comment]
Thanks for your review @marounkoussaifi , I applied the changes.

File: src/docs/admin/references/ServiceNowReference.adoc [hunk 4]
```diff
@@ -101,461 +101,11 @@ image::servicenow-search-by-status.png[align="center"]
 Result should be empty.
 
 
-==== Communicate with ProActive
+==== Usage
 
-Once the application is installed, you are ready to integrate ProActive in your ServiceNow pipelines.
+Once the application is imported in your ServiceNow instances, you are ready to communication both ways between ProActive and ServiceNow.
```
Comment (aminelouati) [inline src/docs/admin/references/ServiceNowReference.adoc:106]: application repository/file/archive/artifact/? 
the two-way communication between ProActive and ServiceNow is ready.

File: src/docs/admin/references/ServiceNowReference.adoc [hunk 5]
```diff
@@ -101,461 +101,11 @@ image::servicenow-search-by-status.png[align="center"]
...
-This window automatically opens when the activity is dragged and dropped on the canvas.
-
-Use ProActive's custom activities from the Custom tab to integrate ProActive in your pipelines.
-
-Here an example of a simple Workflow, that can be stored and used as a Sub Workflow, that Submits a job to ProActive from the Catalog
-
-image::servicenow-workflow-example.png[align="center"]
\ No newline at end of file
+- The available ServiceNow workflows in ProActive's catalog
+- How to use ProActive client in ServiceNow
```
Comment (aminelouati) [inline src/docs/admin/references/ServiceNowReference.adoc:111]: in ServiceNow instance

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 6]
```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
...
+=== ServiceNow
+
+The ServiceNow connectors allow ProActive to communicate with ServiceNow instances to perform CRUD operations on their data as well as to send emails.
+
+The installation of the ProActive application in ServiceNow, which enables a ServiceNow instance to communication with a ProActive server, is described
+in the link:../admin/ProActiveAdminGuide.adoc#_servicenow_integration[ServiceNow admin documentation].
+
+==== ProActive to ServiceNow
+
+A set of ProActive workflows are available in the catalog to communicate with ServiceNow.
```
Comment (aminelouati) [inline src/docs/user/references/ApplicationConnectors.adoc:93]: A set of ProActive workflow templates

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 7]
```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
...
+| e.g. administrator
+| `PASSWORD`
+| ServiceNow password.
+| Yes
+| Sring (HIDDEN)
+| e.g. PWD
+| `TASK_TYPE`
+| The subtype of the ServiceNow Task
+| Yes
+| String
```
Comment (aminelouati) [inline src/docs/user/references/ApplicationConnectors.adoc:208]: is it possible to have PA:LIST() here ?

File: src/docs/user/references/ApplicationConnectors.adoc (hunk 6 above)
Comment (aminelouati) [reply src/docs/user/references/ApplicationConnectors.adoc:93]: Regarding the usage of these connectors, may be mention that they are generally integrated as an activity task in a end-to-end workflow

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 8]
```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
...
+|The session id associated to the user
+
+|restartInErrorTasks
+|Executes a PUT request to restart all tasks in error in the job represented by a job id
+a|
+* jobId: Id of the job to restart in error tasks
+|True if success, false if not
+
+|submitJobFromCatalog
+|Submits a workflow to the scheduler from a workflow URL, creating a new job resource
```
Comment (aminelouati) [inline src/docs/user/references/ApplicationConnectors.adoc:457]: Submits a workflow to the scheduler --> Submits a job to the scheduler
Could you give an example of workflow URL?

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 9]
```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
...
+a|
+* proactiveUrl: Root URL of ProActive server
```
Comment (aminelouati) [inline src/docs/user/references/ApplicationConnectors.adoc:459]: What does it mean Root URL?

File: src/docs/admin/references/ServiceNowReference.adoc (hunk 4 above)
Comment (AlexFalessi) [reply src/docs/admin/references/ServiceNowReference.adoc:106]: Sorry I don't understand the `application repository/file/archive/artifact/?`

File: src/docs/user/references/ApplicationConnectors.adoc (hunk 7 above)
Comment (AlexFalessi) [reply src/docs/user/references/ApplicationConnectors.adoc:208]: It is actually a List. I'll update the type here. Thanks

File: src/docs/user/references/ApplicationConnectors.adoc (hunk 8 above)
Comment (AlexFalessi) [reply src/docs/user/references/ApplicationConnectors.adoc:457]: Im going to rephrase into `Submits a workflow to the scheduler from the catalog, creating a new job resource`
as it is confusing.

You cannot actually provide a URL of a proactive workflow. Instead you specify a bucket and a workflow name that will be fetched from the catalog.

File: src/docs/user/references/ApplicationConnectors.adoc (hunk 9 above)
Comment (AlexFalessi) [reply src/docs/user/references/ApplicationConnectors.adoc:459]: I'm going to switch with `URL of the ProActive installation folder`

Comment (aminelouati) [reply src/docs/user/references/ApplicationConnectors.adoc]: Could you check all occurrences please?

**Review by aminelouati — APPROVED** [review]

**Pull request merged**